docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,77 @@
-# Configuring Zowe certificates 
+# Zowe certificate configuration overview
 
-As a system administrator, review this article to learn about the key concepts of Zowe certificates.
+As a system programmer, review this article to learn about the key concepts of Zowe certificates, and options for certificate configuration. 
 
-Zowe uses a certificate to encrypt data for communication across secure sockets. The certificate used is specified in the `zowe.certificates` section of the `zowe.yaml`, and can either be a `PKCS12` certificate held in a USS keystore `.p12` file, or else a `JCERACKS` certificate held in a keyring.  
+Zowe uses digital certificates to verify the identity and subsequently establish an encrypted network connection between applications using the Secure Sockets Layer/Transport Layer Security (SSL/TLS) protocol. Certificates and their associated private keys need to be stored either in the SAF key ring or in the `PKCS12` java keystore.
```

Comment:
"digital certificates". Never heard of certificates said this way. This sentence is way too technical for me actually, I felt overwhelmed reading it.